docs/sources/administration/data-source-management/teamlbac/configure-teamlbac-for-prometheus/index.md

```diff
@@ -57,7 +57,7 @@ For more information on how to setup LBAC for data sources rules for a Prometheu
 
 ## Grafana Enterprise
 
-LBAC for data sources is available in Grafana Enterprise for Prometheus created with basic authentication. Prometheus data sources for LBAC for data sources can only be created.
+LBAC for data sources is available in Grafana Enterprise for Prometheus connected to GEM created with basic authentication. Prometheus data sources connected with GEM for LBAC for data sources can only be created.
```

Comment:
I'm not sure what the second sentence is saying here. Is it about provisioning? If so, something like `Data source provisioning is not supported.`

Reply:
I think this was too much info for what i wanted to say. removed that sentence